doc/01-usage.md

```diff
@@ -149,6 +149,103 @@ error: options validation failed for image type "iot-raw-xz": ostree.url: requir
 $
 ```
 
+### bootc
+
+`image-builder` supports building images from [bootable containers](https://docs.fedoraproject.org/en-US/bootc/getting-started/). Building bootc-based images works differently from `ostree`-based images and package-based images.
+
+When building a bootable container into an image we try to base everything on the container. Thus the distribution that is being built is not known; you cannot use the `--distro` argument in combination with `--bootc-*` arguments as it would do nothing.
```

Comment:
I'd drop `try to` since that implies that there may be cases where it isn't based on the container.

Reply:
There are things that aren't based on the container; for example customizations and we do use (some) host tooling such as `skopeo` I believe.